docker/prepare-dev-environment/volumes/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.6"
+
+services:
+  volumes-populator:
+    build: ${DOCKERFILE_PATH}
+    volumes:
+      - atlas-data-bioentity-properties:/atlas-data/bioentity_properties
+      - atlas-data-exp:/atlas-data/exp
+    environment:
+      EXP_IDS: ${EXP_IDS}
```

Comment:
Do we need to redefine the env variable that was sourced from the dev.env file?

Reply:
Removed!